ehitchhiking-ui/src/app/shared/components/chat/chat.component.sass

```diff
@@ -0,0 +1,132 @@
+@import "../../../../styles/colors"
+@import "../../../../styles/mixins"
+@import "../../../../styles/fonts-constants"
+@import "../../../../styles/constants"
+$image-size: 2rem
+$before-horizontal-radius: 0.5rem
+$before-vertical-radius: 0.4rem
+$message-border-width: 0.625rem
+$after-horizontal-radius: 0.3rem
+$after-width: 0.8rem
+$before-right: -0.2rem
+
+.modal-wrapper
+  .chat
+    .message-zone
+      overflow-x: hidden
+      height: 20rem
+      max-width: 23rem
+      margin: $default-offset/2
+      font-size: $body-2
+      font-weight: normal
+      display: flex
+      flex-direction: column
+.input-zone
+  display: flex
+  justify-content: space-between
+  .input
+    width: 40rem
+  .send-btn
+    background-color: mat-color($primary, 50)
+    height: 2.8rem
+.modal-back
+  position: absolute
+  top: 0.5rem
+  left: 0.5rem
+  @include flexbox()
+  color: mat-color($primary, 400)
+  &-btn:hover
+    background-color: mat-color($accent, 50)
+  &-btn
+    .btn-text
+      display: inline
+
+
+.message-wrapper
+  display: flex
+  justify-content: flex-end
+  .message
+    flex-direction: column
+    flex-wrap: wrap
+    .dlg-title
```

Comment:
styles for dialog-list should not be here

Reply:
classes renamed